packages/rum-core/src/domain/vital/vitalCollection.ts

```diff
@@ -74,9 +98,9 @@ function processVital(vital: DurationVital, valueComputedBySdk: boolean): RawRum
       id: generateUUID(),
       type: vital.type,
       name: vital.name,
-      custom: {
-        [vital.name]: vital.value,
-      },
+      duration: vital.duration,
+      details: vital.details,
+      custom: {},
```

Comment:
❓ **Question**: why not opting for making custom nullable ?

Reply:
Yes, why do we still need `custom`?

Reply:
We don't. I'll remove it.